bodhi/tests/server/services/test_updates.py

```diff
@@ -163,7 +163,14 @@ def test_no_privs(self, publish, *args):
         app = TestApp(main({}, testing=u'bodhi', session=self.db, **self.app_settings))
         res = app.post_json('/updates/', self.get_update(u'bodhi-2.1-1.fc17'),
                             status=400)
-        assert 'bodhi does not have commit access to bodhi' in res, res
+        expected_error = {
+            "location": "body",
+            "name": "builds",
+            "description": ("bodhi is not a member of \"packager\", which is a"
+                            " mandatory packager group")
+        }
+        assert expected_error in res.json_body['errors'], \
+            res.json_body['errors']
```

Comment:
Optional: It might be nice to use ```self.assertTrue()``` here, but I can't think of a particularly strong advantage to one over the other.